devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -113,6 +113,12 @@ def refresh() {
     zigbee.onOffRefresh() + zigbee.levelRefresh()
 }
 
+def installed() {
+    if ((device.getDataValue("manufacturer") == "MRVL") && (device.getDataValue("model") == "MZ100")) {
```

Comment:
I would add a comment reflecting the fact that this is to accommodate a FW defect and what the case would be for removing this if-block.